test/unit/test.step.js

```diff
@@ -7,24 +7,27 @@ const { assert } = chai;
 import Shepherd from '../../src/js/shepherd.js';
 import { Step } from '../../src/js/step.js';
 import { Tour } from '../../src/js/tour.js';
+import tippy from 'tippy.js';
```

Comment:
Shouldn't we be importing Tippy in one of the other files above we are already importing?

Reply:
I'm not sure I'm following. Other files might be importing it, but I don't think that would make it global if that's what you mean?

Reply:
I think I was thinking perhaps it was added onto `Shepherd` for instance, to access via `Shepherd.tippy`. If it's not available elsewhere, this is fine 👍